ngraph/test/visitors/op/normalize_l2.cpp

```diff
@@ -18,11 +15,11 @@ using namespace ngraph;
 using ngraph::test::NodeBuilder;
 using ngraph::test::ValueMap;
 
-TEST(attributes, normalize_l2_op)
+TEST(attributes, normalize_l2_op_mode_add)
```

Comment:
Please consider to use this approach to reduce code duplication: #6287

Reply:
Like it, done. Please take look at the updated version.